response_operations_ui/forms.py

```diff
@@ -242,3 +242,43 @@ def validate_password(form, field):
         if password.isalnum() or not any(char.isupper() for char in password) or not any(char.isdigit() for char in
                                                                                          password):
             raise ValidationError('Your password doesn\'t meet the requirements')
+
+
+class RequestAccountForm(FlaskForm):
+    '''The form used by request-new-account.html'''
+    email_address = StringField('Enter the ONS email address to create an account for',
+                                validators=[InputRequired('Enter an email address'),
+                                            Email(message='Invalid email address'),
+                                            Length(max=254,
+                                                   message='Your email must be less than 254 characters')])
+    password = PasswordField('Enter the admin password',
+                             validators=[InputRequired('Enter the admin password')])
+
+    @staticmethod
+    def validate_email_address(_, field):
+        email = field.data
+        _validate_email_address(email)
+        local_part, domain_part = email.rsplit('@', 1)
+        if domain_part != 'ons.gov.uk' and domain_part != 'ext.ons.gov.uk' and domain_part != 'ons.fake':
```

Comment:
Missed this on the first time around.  A cleaner way to do it would be:
```suggestion
        if domain_part not in ['ons.gov.uk', 'ext.ons.gov.uk', 'ons.fake']:
```

Second, the ons.fake part of this is a testing domain that could change at any time.  I think it's wrong to hard code test domain in this but I'm not sure what's correct.  Maybe put it in the testing config?  Have a think, it's wrong but I'm not gonna block the review on this cause I don't know what right looks like